scripts/osx/testTemplate.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+OF_DIR=`pwd`;
+echo $OF_DIR;
+TEST_DIR="$OF_DIR/scripts/osx"
+cd $TEST_DIR
```

Comment:
do we need all that directory hopping? can't you supply a path to xcodebuild (i don't know), so only would have to execute xcodebuild in the build step and be done with it (and get rid of this script)?